src/Seed.php

```diff
@@ -491,7 +495,11 @@ public static function markupFromRoute($route, array $headers = []) {
     $headers['Host'] = $hostname;
 
     // Generate a signed token and use it in the request.
-    $headers['quant-token'] = \Drupal::service('quant.token_manager')->create($route);
+    // This only applies when drafts are enabled, as we return neutral access otherwise.
+    $disable_drafts = $config->get('disable_content_drafts');
+    if (!$disable_drafts) {
+      $headers['quant-token'] = \Drupal::service('quant.token_manager')->create($route);
+    }
```

Comment:
With the `if (!$disable_drafts) {` logic in place, I can't get published content to sync at all when `Disable content drafts` is enabled. I tested this with both Workbench Moderation and Content Moderation modules uninstalled.

Reply:
Latest commit 2bca4a7 will resolve by disabling token verification when drafts are disabled -- glad you picked that up!